### ISSUE TITLE
Fails at the 8080 port check

### ISSUE THREAD

**Issue by tscolari** [githubixx/ansible-role-kubernetes-controller#11]
Hey there, I've updated my cluster recently and it seems to get stuck on this:

```
- name: Wait for kube-apiserver to listen on port 8080 on the host
  wait_for:
    port: 8080
    delay: 5
    host: "{{hostvars[inventory_hostname]['ansible_' + k8s_interface].ipv4.address}}"
  run_once: true
  delegate_to: "{{groups.k8s_controller|first}}"
```

I wonder if this has anything to do with 

```
removed deprecated kube-apiserver parameter insecure-bind-address (see: #59018)
``` 

from your blog post changelog.
kube-apiserver seems to be binding :8080 to localhost for me, so this check is never successful.

**Comment by githubixx** [comment]
Hmmm... That's indeed strange that this task still exists. The insecure-port is deprecated so this check don't make sense (in contrast to the check for port 6443). So I would try to just remove the task for now. Will have a look later. Just wondering why it worked for me while it shouldn't :wink:

**Comment by tscolari** [comment]
Yeah, I thought so too! Also weird, because it worked the first time, then I made some extra modifications and it started failing. Took a while to narrow it down to this, commenting this out made it through.

I had it deployed a version before you introduced the controller configuration after I commented this check out, generated the missing configs and certs things did deploy. I can create pods and etc. but things like exec,port-forward,etc gives a permission error:

```
error: error upgrading connection: unable to upgrade connection: Forbidden (user=Kubernetes, verb=create, resource=nodes, subresource=proxy)
```

also the journalctl for the kube-apiserver keeps complaining about certs:
```
Jun 29 13:09:18 k8s-node-1 kube-apiserver[10598]: E0629 13:09:18.232042   10598 authentication.go:62] Unable to authenticate the request due to an error: [invalid bearer token, [invalid bearer token, square/go-jose: error in cryptographic primitive]]
```
even though most of the things work.

Is there any special thing that has to be done when migrating from one version of the doc to the other?
